src/disrupt.sh

```diff
@@ -22,6 +23,12 @@ if [[ ! -d "${IMG_ASSET_DIR}" || ! "${IMAGES_LIST}" ]]; then
     DICE_ROLL=0
 fi
 
+IMAGE_MATCH="${IMG_ASSET_DIR}/$($IMAP "$RANDOM_FACT")"
+if [[ -f $IMAGE_MATCH ]]; then
+    DICE_ROLL=1
+    RANDOM_IMAGE=$IMAGE_MATCH
```

Comment:
Maybe this shouldn't be called `RANDOM_IMAGE` if it's being used for both the random image and the associated images.